packages/react-codemirror/playwright-ct.config.ts

```diff
@@ -14,9 +14,9 @@ export default defineConfig({
   /* Fail the build on CI if you accidentally left test.only in the source code. */
   forbidOnly: !!process.env.CI,
   /* Retry on CI only */
-  retries: process.env.CI ? 2 : 0,
-  /* Opt out of parallel tests on CI. */
-  workers: process.env.CI ? 1 : undefined,
+  retries: 2,
+  /* Use parallel testing */
+  workers: undefined,
```

Comment:
Fully parallel isn't recommended on CI unless you have powerful machines, as you're already using much of the computing power. We can measure, but I think the stability vs. time computation isn't there unless we switch to a larger machine, see this github issue asking about the default on their page 

https://github.com/microsoft/playwright/issues/22666

Reply:
I don't think it was making the jobs faster anyway because our GitHub machines only have 2 cores